app/models/user.rb

```diff
@@ -0,0 +1,32 @@
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  attr_reader   :password
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.